benchmarks/ClusterMicroBenchmarks/ConcurrentSpawnBenchmark.cs

```diff
@@ -96,7 +96,19 @@ public void RemoveActivations()
             tasks[i] = _cluster.RequestAsync<Terminated>(id, PoisonPill.Instance, cts.Token);
         }
 
-        Task.WhenAll(tasks).GetAwaiter().GetResult();
+        try
+        {
+            Task.WhenAll(tasks).GetAwaiter().GetResult();
```

Comment:
This will be a change in behaviour.. but not sure it's a problem in this context

Reply:
Not sure what you mean. The previous version ignored timeouts and the catch I added does is meant to reproduce this behavior with the new version. 

Overall the behavior of Benchmark.NET is a bit weird when exception is thrown in the cleanup method. It seems to ignore it but then benchmark  crashes on super weird issues.

Reply:
The block ignores it, but it does not wait for the remaining tasks? Previously each call "suppressed" the timeout, now it should throw on the first failing call?

Reply:
Anyways, it's a benchmark, not critical path :)

Not too concerned if the semantics changed a little there

Reply:
Task.WhenAll waits for all tasks even if one of the fails. Am I missing something here?

Reply:
Ping @mhelleborg 
Should I just resolve this?